tests/hazmat/asn1/test_serialization.py

```diff
@@ -2,14 +2,27 @@
 # 2.0, and the BSD License. See the LICENSE file in the root of this repository
 # for complete details.
 
+import typing
+
 import pytest
 
 import cryptography.hazmat.asn1 as asn1
 
 
-class TestEncodeInteger:
+def assert_roundtrip(obj: typing.Any, obj_bytes: bytes) -> None:
+    encoded = asn1.encode_der(obj)
+    assert encoded == obj_bytes, (
+        f"Failed for {obj}: got {encoded.hex()}, expected{obj_bytes.hex()}"
+    )
```

Comment:
We don't generally use assertion messages -- they're pretty duplicative of what pytest already does, and strictly worse than just getting in there with a pdb. (In this case, I personally almost always prefer looking at the raw bytes rather than the hex, for example.)

Reply:
fixed